build/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/lvl484/positioning-filter/positioning-filter/build
```

Comment:
What do you think about renaming module name to 'github.com/lvl484/positioning-filter'?

Reply:
Yep, have changed that)